package.json

```diff
@@ -27,7 +27,7 @@
   },
   "homepage": "https://github.com/SamVerschueren/gulp-cordova-build-android",
   "dependencies": {
-    "cordova-lib": "^6.0.0",
+    "cordova-lib": "^6.1.1",
```

Comment:
not necessary because of `semver`. It will automatically install the latest `6.x` version when running `npm install gulp-cordova-build-android`. So you can remove it back.